config.toml.example

```diff
@@ -334,8 +330,8 @@
 # This is an array of the codegen backends that will be compiled for the rustc
 # that's being compiled. The default is to only build the LLVM codegen backend,
 # but you can also optionally enable the "emscripten" backend for asm.js or
-# make this an empty array (but that probably won't get too far in the
-# bootstrap)
+# make this an empty array (that will disable LLVM, but bootstrap will fail at
+# the time the final rustc binary is built for stage 1)
```

Comment:
Is this the right stage? I always get confused about it.

Reply:
I don't think this needs to be that explicit, keeping the original "but that probably won't get too far in the bootstrap" and just adding "which will disable LLVM" before it seems enough.